CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
 * Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
 * Address book working much better than before
+* Fixed Invalid bookmark error when clicking on 'All address lists' entry in recipient selection dialog box
```

Comment:
We are using markdown so it would be awesome if you change the entry slightly using `` to something like:

    Fixed `Invalid bookmark` error when clicking on `All address lists` entry in recipient selection dialog box